stages/release.md

```diff
@@ -6,12 +6,11 @@
 ### __Stage description:__  
 Insert your description of the stage here  
 
-| Pipeline Stage:<br>Release  | Role 1  | Role 2  | Role 3  | Role 4  | Role N  |
+| Pipeline Stage:<br>Release  | Product Owner  | Client  | Developer  | Senior Manager  | QA Engineer  |
 |----------------------------- |-------- |-------- |-------- |-------- |-------- |
-| Task 1                       |         |         |         |         |         |
-| Task 2                       |         |         |         |         |         |
-| Task 3                       |         |         |         |         |         |
-| Task N                       |         |         |         |         |         |
+| Release to QA                |    I     |        |   AC     |         |    RV     |
+| Release to Staging if tests pass    |    I     |         |   RA     |    I     |    V    |
```

Comment:
```suggestion
| Release to Staging if tests pass    |    I     |         |   RA     |    I     |    VA    |
```
Here I'd only add and `Accountable` role to the QA Engineer, since they will be doing some work there and block the release going forward.

Reply:
Good call, agreed.